types/react-router-redux/index.d.ts

```diff
@@ -25,15 +25,15 @@ export interface ConnectedRouterProps<State> {
 }
 export class ConnectedRouter<State> extends React.Component<ConnectedRouterProps<State>> {}
 
-export const LOCATION_CHANGE: string;
+export const LOCATION_CHANGE: '@@router/LOCATION_CHANGE';
```

Comment:
At the moment we expose only types for everything we export from d.ts file.

But you want to expose values for two constants.

I cannot anything bad about it except it's inconsistency with all the other code.

Can you, please, give me a reason, why should we do it this way? What benefits do we gain in price of this consistency?

If some variable can have two possible values, we'd write `const position: 'left' | 'right';`. But when there is only one possible value, you want to use assignment operator instead.